fusor-ember-cli/app/models/openstack-deployment.js

```diff
@@ -39,6 +39,8 @@ export default DS.Model.extend(ValidatedModel, {
   undercloud_ssh_username: DS.attr('string'),
   undercloud_ssh_password: DS.attr('string'),
 
+  overcloud_deployed: DS.attr('boolean'),
```

Comment:
I would like to propose that I just go back and make sure I'm using ? appended on the back (obj.overcloud_deployed?).  It's unambiguous and in keeping with Ruby/Rails norms.

I feel like prefixing things with "is_" would be a big strange for database fields.  I've only got experience with Ruby and Java, but in both those I think it's more normal to leave the is_ off.

For Ruby, most conditionals look like (.empty?, nil?, .any?, etc) and Rails knows to allow the question mark.  [stackoverflow](http://stackoverflow.com/questions/3112078/rails-boolean-fields-is-foo-or-just-foo)

For Java getters and setters for booleans always start with "is" but the database fields don't or else you'd get crazy names ilke "isIsOvercloudDeployed()".